### DISCUSSION TITLE
forced arm channel

### DISCUSSION THREAD

**Discussion by wx4cb** [ExpressLRS/ExpressLRS#1377] [Ideas]
it would be nice to not be forced to use channel 5 (aux1) as an arm switch.

For those of use that are flying planes with ELRS and not using a flight controller, being forced to use channel 5 as an arm switch is stupid.

for example, i have a 3 channel glider that has a pan/tilt on it. using the matek PWM receiver, i'm forced to either have the one output on channel 5 as a high/low or move that output to channel 6. Throttle  cut/arming is handled by open TX.

so now my channel lay out is  throttle, aileron, rudder, pan, disabled, tilt.

It would be nice to be able to have a flag in the configurator to disable "force channel 5 to be high/low".. Note i'm using the "wide" settings.

**Comment by AlessandroAU** [comment]
For the PWM RX you can remap any of the other channels to output 5 instead of AUX1

**Reply by wx4cb** [reply]
> For the PWM RX you can remap any of the other channels to output 5 instead of AUX1

correct, but you are still losing a physical channel that otherwise would be usable.

**Reply by FOG-Yamato** [reply]
The arming flag from AUX1 is responsible for multiple features within the ELRS code and as such it can't be changed that easily.

**Comment by wx4cb** [comment]
there's a myriad of other ways to achieve it without being forced to lose a channel for it, heck... you could just add it so that whatever channel is being used for arm (and can be assigned in the configurator at compile time) should be what's used. you can still have your default channel as five, but allow it to be configurable at compile time

**Reply by avsaase** [reply]
How do you lose a channel? You have channels 1-4 for controls. Channel 5 has only two positions so you will never get a fifth full resolution channel.

**Reply by wx4cb** [reply]
> How do you lose a channel? You have channels 1-4 for controls. Channel 5 has only two positions so you will never get a fifth full resolution channel.

but that's my point... there's no reason for it to be forced as channel 5.... set it up in the configurator that you can specify your arming channel...

**Reply by avsaase** [reply]
But how does that matter for your use case? You can remap the channels to the output pins on the PWMP receiver however you want.

**Comment by DutchFPV** [comment]
On this same topic, is stick arming affected?  I have stick armed without incident since 2015.  Its fully calibrated in my brain and changing to a switch would only introduce risk, mostly to me.  Is it safe to assume that there's no concern if I'm not designating any switch as ARM?
